### PR TITLE
feat(tabs): provide first version of navigation tabs - INNO-600

### PR DESCRIPTION
**Pull request by yhuard**
What should be the responsive behavior? Are there other variants (e.g. pills)?

TODO:
- [x] write tests
- [x] use standard variables

### CODE REVIEW

File: framework/components/ecl-navigation/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Navigation
+
+## Default
+
+## Tabs
```
Comment (emeryro) [inline framework/components/ecl-navigation/README.md]: Should be a little more descriptive

File: framework/components/ecl-navigation/_navigation.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.ecl-navigation__item {
+  flex-shrink: 0;
+
+  .ecl-navigation--tabs & {
+    margin-bottom: -$ecl-navigation-border-width;
+  }
+}
+
+.ecl-navigation__item-link {
```
Comment (emeryro) [inline framework/components/ecl-navigation/_navigation.scss]: we have to synchronize here: I named it __link in the horizontal menu, and not __item-link.
So which one do we keep?
I would go for __link as it is descriptive enough

File: framework/components/ecl-navigation/_navigation.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  display: block;
+  font-weight: bold;
+  padding: map-get($ecl-spacing, 'xs') map-get($ecl-spacing, 's');
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  /* stylelint-disable-next-line */
```
Comment (emeryro) [inline framework/components/ecl-navigation/_navigation.scss]: Why? :)

File: framework/components/ecl-navigation/_navigation.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  .ecl-navigation--tabs & {
+    border: $ecl-navigation-border-width solid map-get($ecl-colors, 'grey-50');
+    border-bottom-color: #fff;
+    text-decoration: none;
+
+    /* stylelint-disable-next-line */
+    &:focus {
+      background-color: $ecl-color-secondary;
```
Comment (emeryro) [inline framework/components/ecl-navigation/_navigation.scss]: Not sure we need to re-declare background-color here

Comment (yhuard) [inline framework/components/ecl-navigation/_navigation.scss]: Good catch 👍

File: framework/components/ecl-navigation/_navigation.scss (hunk 2 above)
Comment (yhuard) [inline framework/components/ecl-navigation/_navigation.scss]: Hhmmm, we need to sync. The horizontal menu is a navigation component

**Pull request merged**